pkg/scanner/local/scan_test.go

```diff
@@ -301,39 +354,43 @@ func TestScanner_Scan(t *testing.T) {
 				},
 			},
 			fixtures: []string{"testdata/fixtures/happy.yaml"},
-			applyLayersExpectation: ApplierApplyLayersExpectation{
-				Args: ApplierApplyLayersArgs{
-					BlobIDs: []string{"sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10"},
-				},
-				Returns: ApplierApplyLayersReturns{
-					Detail: ftypes.ArtifactDetail{
-						OS: ftypes.OS{
-							Family: ftypes.Alpine,
-							Name:   "3.11",
+			setupCache: func(t *testing.T) cache.Cache {
+				c := cache.NewMemoryCache()
+				require.NoError(t, c.PutBlob("sha256:5216338b40a7b96416b8b9858974bbe4acc3096ee60acbc4dfb1ee02aecceb10", ftypes.BlobInfo{
+					SchemaVersion: ftypes.BlobJSONSchemaVersion,
+					OS: ftypes.OS{
+						Family: ftypes.Alpine,
+						Name:   "3.11",
+					},
+					PackageInfos: []ftypes.PackageInfo{
+						{
+							FilePath: "lib/apk/db/installed",
+							Packages: []ftypes.Package{muslPkg},
 						},
-						Packages: []ftypes.Package{
-							muslPkg,
-							python39min,
+						{
+							FilePath: "usr/lib/python/site-packages/urllib3-3.2.1/METADATA",
+							Packages: []ftypes.Package{urllib3Pkg},
```

Comment:
Previous test included `python39min`:
https://github.com/aquasecurity/trivy/blob/3e503a0cc24c4e35366f4813fda67ef654f33c1f/pkg/scanner/local/scan_test.go#L316

Add this package here or remove [this variable](https://github.com/aquasecurity/trivy/blob/06ccb8f92685474635d5d2f0d1795efc8bf892a0/pkg/scanner/local/scan_test.go#L99-L107) (because it is not used):